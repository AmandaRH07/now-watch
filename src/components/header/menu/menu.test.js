import { BrowserRouter } from "react-router-dom"
import { render } from "@testing-library/react"
import MenuOptions from "."

const renderComponent = () => (
  render(
    <BrowserRouter>
      <MenuOptions />
    </BrowserRouter>
  )
)

describe('<MenuOptions/>', () => {

  it('Should render MenuOptions', () => {
    const {container} = renderComponent();

    expect(container).toMatchSnapshot();
  })
})