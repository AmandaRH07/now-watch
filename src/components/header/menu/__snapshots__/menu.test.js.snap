// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MenuOptions/> Should render MenuOptions 1`] = `
<div>
  <button
    aria-haspopup="true"
    class="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium sc-aXZVg iHibrM css-1e6y48t-MuiButtonBase-root-MuiButton-root"
    id="basic-button"
    tabindex="0"
    type="button"
  >
    Menu
    <span
      class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
    />
  </button>
</div>
`;
